rcl_yaml_param_parser/src/parse.c

```diff
@@ -585,6 +594,166 @@ _validate_name(const char * name, rcutils_allocator_t allocator)
   return ret;
 }
 
+rcutils_ret_t
+_validate_tag(const char * tag, uint32_t line_num)
+{
+  if (tag == NULL) {
+    return RCUTILS_RET_ERROR;
+  }
+
+  if ((0 == strcmp(tag, YAML_NULL_TAG)) ||
+    (0 == strcmp(tag, YAML_BOOL_TAG)) ||
+    (0 == strcmp(tag, YAML_STR_TAG)) ||
+    (0 == strcmp(tag, YAML_INT_TAG)) ||
+    (0 == strcmp(tag, YAML_FLOAT_TAG)) ||
+    (0 == strcmp(tag, YAML_TIMESTAMP_TAG)) ||
```

Comment:
Now, TIMESTAMP tag is supported ?

Reply:
No, it's a mistake, I copyed the tags in batch, I have removed it.